Eatery/Controllers/EateryTabBarController.swift

```diff
@@ -15,26 +15,26 @@ class EateryTabBarController: UITabBarController {
 
     // MARK: View controllers
 
-    let eateriesSharedViewController = EateriesSharedViewController()
+    let eateriesViewController = CampusEateriesViewController()
     let lookAheadViewController = LookAheadViewController()
     let brbViewController = BRBViewController()
 
     override func viewDidLoad() {
         delegate = self
-        eateriesSharedViewController.tabBarItem = UITabBarItem(
+        eateriesViewController.tabBarItem = UITabBarItem(
             title: nil,
             image: UIImage(named: "eateryTabIcon.png"),
             tag: 0
         )
-
+        
         let lookAheadNavigationController = EateryNavigationController(rootViewController: lookAheadViewController)
         lookAheadNavigationController.tabBarItem = UITabBarItem(title: nil, image: #imageLiteral(resourceName: "menu icon"), tag: 1)
 
         let brbNavigationController = EateryNavigationController(rootViewController: brbViewController)
         brbNavigationController.tabBarItem = UITabBarItem(title: nil, image: UIImage(named: "accountIcon.png"), tag: 2)
 
         let navigationControllers = [
-            eateriesSharedViewController,
+            eateriesViewController.campusNavigationVC,
```

Comment:
I think a cleaner design would be to have the navigation controller be initialized in this method, then have the `EateryTabBarController` implement `UINavigationControllerDelegate` and move the methods for showing and hiding the navigation bar into here.

Reply:
if there's no issues with the approach that Will is suggesting, i think that'd be cleaner as well!

Reply:
This originally resulted in a logic error even after implementing the delegate.  I'll try to work around this though

Reply:
This did result in a run-time error but I cant try to work around this

Reply:
What runtime error? Can you give more details about why it didn't work?